### ISSUE TITLE
Organization Edit - Error state text color

### ISSUE THREAD

**Issue by andyarensman** [hackforla/food-oasis#1703]
### Overview
When there is an error in a text field on the organization edit form, the text should be red.

### Action Items
- [x] change the color of the placeholder from grey to red when there is an error
- [x] check to see if any other text needs to change to red

### Resources/Instructions
- See the [Figma Design #2](https://www.figma.com/file/9hPmNSsyaH2VMGNz5mffmI/FOLA-Design-%232?type=design&node-id=3774-4672&t=e8doaoTUsWls0rhJ-0) for more info. There is a purple post it note for the issue.

**Comment by andyarensman** [comment]
The placeholder text has been changed to red when there is an error. I also fixed some spacing issues with the drop down helper text and made the phone and email properly display the error message.

![chrome_ddodlccir1](https://github.com/hackforla/food-oasis/assets/86622025/755f793a-01c3-4b87-aef9-ddc6b082da5a)

![chrome_KMXl2UX2SV](https://github.com/hackforla/food-oasis/assets/86622025/53715bf8-4dd7-4196-93bf-8f7075207d1d)

**Comment by andyarensman** [comment]
Note: Phone and email are no longer being required

**Comment by entrotech** [comment]
@andyarensman Testing this in the dev environment. When a field is required, it makes sense to change the color of the placeholder, since there is no text in the field, but what if a different rule is violated? For example, if the email is not in a valid format, maybe the text should be in red? Right now the text doesn't change color when other validation rules are broken:
![image](https://github.com/hackforla/food-oasis/assets/9939032/a56cb422-e488-42f4-8fc3-b3dadb8aa4ca)


Need to decide whether the text should change when the input breaks any validation rule, and create a follow-up issue, if needed to implement this as well.

**Comment by andyarensman** [comment]
@entrotech I will look into this and create a new issue
